src/main/java/net/dries007/tfc/client/render/entity/CougarRenderer.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed under the EUPL, Version 1.2.
+ * You may obtain a copy of the Licence at:
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ */
+
+package net.dries007.tfc.client.render.entity;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import net.dries007.tfc.client.RenderHelpers;
+
+import net.dries007.tfc.client.model.entity.CougarModel;
+import net.dries007.tfc.common.entities.aquatic.Jellyfish;
+import net.dries007.tfc.common.entities.predator.Cougar;
+import net.minecraft.client.renderer.entity.EntityRendererProvider;
+import net.minecraft.client.renderer.entity.MobRenderer;
+import net.minecraft.resources.ResourceLocation;
+
+public class CougarRenderer extends MobRenderer<Cougar, CougarModel>
+{
+    private static final ResourceLocation COUGAR_LOCATION = RenderHelpers.animalTexture("cougar");
+
+    public CougarRenderer(EntityRendererProvider.Context ctx)
+    {
+        super(ctx, new CougarModel(ctx.bakeLayer(RenderHelpers.modelIdentifier("cougar"))), 0.9F);
+    }
+
+    @Override
+    protected void scale(Cougar cougar, PoseStack poseStack, float ticks)
+    {
+        poseStack.scale(1.2F, 1.2F, 1.2F);
```

Comment:
i would prefer the actual mdoel is fixed here rather than hardcoding this but it doesn't matter that much

Reply:
further, there should after that be no Cougar class, just the regular feline class that gets instantiated twice

Reply:
Okay, I think this is just something I copied over from the polar bear class without understanding it, but I can look into changing it.

As far as the Cougar class, you are saying I should just add cougars and panthers as two separate entity types rather than one with two variants? Or are you saying the variant code should be in the BigCat class somehow? I just did it this way because I wasn't sure of a good way to make the panther variant rare if they were separate mobs. And while it isn't that important for panthers to be rare, I am kinda attached to the idea.

Reply:
They should be different entity types. We can work on rarity at another point (there's a lot of levers to pull)